.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+Before submitting an issue, please check over the [issue tracker](https://github.com/diesel-rs/diesel/issues) to see if your problem has been discussed before.
```

Comment:
This is good, but I fear the wall of text my scare people who do not want to report a bug. Maybe we can add something like this at the beginning?

> If you want to report a bug, we added some points below you can fill out. If you want to request a feature, feel free to remove all the irrelevant text. In addition to this issue tracker, you can also talk to Diesel maintainers and users on [Gitter](https://gitter.im/diesel-rs/diesel).